runtime/doc/options.txt

```diff
@@ -3267,7 +3267,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 				     >:SignColumn,B:SpellBad,P:SpellCap,
 				     R:SpellRare,L:SpellLocal,-:Conceal,
 				     +:Pmenu,=:PmenuSel,x:PmenuSbar,
-				     X:PmenuThumb")
+				     X:PmenuThumb,q:QuickFixLine")
```

Comment:
Oh sorry I misread your comment then. I mostly agree, but still think there should be some inline marker in the test for unexpected highlights, like a star, `{*2:[No Name]}`. My solution so far has been to recommend adding a `snapshot_util()` just before the expect but that would certainly be an improvement. (expect for indeterministic failures where `redraw_debug()` is still needed.).